sonar-db/src/main/java/org/sonar/db/DefaultDatabase.java

```diff
@@ -93,7 +97,7 @@ private void initDataSource() throws Exception {
     datasource = new ProfiledDataSource(basicDataSource, NullConnectionInterceptor.INSTANCE);
     datasource.setConnectionInitSqls(dialect.getConnectionInitStatements());
     datasource.setValidationQuery(dialect.getValidationQuery());
-    enableSqlLogging(datasource, "TRACE".equals(settings.getString("sonar.log.level")));
+    enableSqlLogging(datasource, logbackHelper.getLoggerLevel("sql") == Level.TRACE);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Change this comparison to use the equals method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1698)